doc/topics/installation/suse.rst

```diff
@@ -2,14 +2,19 @@
 SUSE Installation
 =================
 
-With openSUSE 13.2, Salt 2014.1.11 is available in the primary repositories.
-The devel:language:python repo will have more up to date versions of salt,
-all package development will be done there.
+Since openSUSE 13.2, Salt 2014.1.11 is available in the primary repositories.
+With the release of suse manager 3 a new repository setup has been created.
+The new repo will by systemsmanagement:saltstack, this repo will be the source
```

Comment:
I would rephrase that to: "The new repo is systemsmanagement:saltstack, which is the source...."